src/utils/grpchelper.cpp

```diff
@@ -19,54 +20,24 @@ using namespace aos;
  **********************************************************************************************************************/
 
 // The PKCS #11 URI Scheme: https://www.rfc-editor.org/rfc/rfc7512.html
-static std::string CreateRFC7512URL(
-    const String& token, const String& label, const Array<uint8_t>& id, const String& userPin)
+static std::string CreateRFC7512URL(const String& url)
```

Comment:
So name of function is not correct. Because we provide URL in RFC7512 format but libp11 doesn't accept it.
So, should be `ConvertToP11URL` or something.

Reply:
done